bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -435,15 +272,14 @@ if lofft_outfile or lofft_h5_outfile:
                     num_trials_louder += 1
         fap = num_trials_louder/total_trials
         pval = '< %.3g' % (1./total_trials) if fap == 0 else '%.3g' % fap
-
-        d = [chunk_num, trig_slide_id, pval,\
-             np.asarray(trig.get_end()).astype(float),\
-             trig.mass1, trig.mass2, trig.mchirp,\
-             (np.degrees(trig.ra)), (np.degrees(trig.dec)),\
-             trig.snr, trig.chisq, trig.bank_chisq,\
-             trig.cont_chisq, trig.get_null_snr()]
-        d.extend([getattr(trig, 'snr_%s' % ifo_att[ifo])\
-                         for ifo in ifos])
+        #bank chisq and cont chisq  replacede with Nones
+        d = [chunk_num, trig_slide_id, pval,
+             np.asarray(trig['end_time']).astype(float),
+             trig['mass1'], trig['mass2'], trig['mchirp'],
```

Comment:
These masses would have to be pulled from the template bank using the `network/template_id` dataset